Automattic/wp-calypso
# Stats: Hides close button on upgrade notice banner

Related to https://github.com/Automattic/red-team/issues/14

## Proposed Changes

* This PR hides the close button on all instances of the Commercial upgrade notice banner

**Followup questions:**

- Do we want this to always be hidden, or just in some instances of the banner being displayed? (eg. Commercial sites under 1K)
- Will this only be a frontend change, or should we block the option in notice API as well?


## Testing Instructions



* On a testing site which is commercial, check for the commercial upgrade notice. Confirm that there is no "X" symbol/way to dismiss this notice. 

## Pre-merge Checklist
